dev-utils/update-tav-versions.js

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict';
+
+// Update version property for each module defined in .tav.yml file
```

Comment:
nit: I think it would be helpful if the top comment here identified the "versions-selection" field as the important one that it is using for this.

Reply:
comment updated